api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/objects/AccessObjectStatement.java

```diff
@@ -0,0 +1,60 @@
+package edu.cornell.mannlib.vitro.webapp.auth.objects;
```

Comment:
Add the license header here
/* $This file is distributed under the terms of the license in LICENSE$ */

Reply:
it is also missing in some other classes in this package

Reply:
fixed